src/pybind/cephfs/cephfs.pyx

```diff
@@ -953,7 +956,7 @@ cdef class LibCephFS(object):
             if ret < 0:
                 raise make_ex(ret, "error in getxattr")
 
-            return ret_buf[:ret]
+            return ret_buf[:ret].decode()
```

Comment:
i don't think we should return a `str` here.

Reply:
Reverted.